src/openrct2/peep/peep.c

```diff
@@ -9816,16 +9830,43 @@ static sint32 guest_path_find_entering_park(rct_peep *peep, rct_map_element *map
 }
 
 /**
+ * Gets the nearest park entrance relative to point, by using manhatten distance.
+ * @param x x coordinate of location
+ * @param y y coordinate of location
+ * @return Index of gPeepSpawns (or 0xFF if no park entrances exist).
```

Comment:
Description of the function is incorrect, replace 'park entrance' with 'peep spawn'